libmamba/include/mamba/fs/filesystem.hpp

```diff
@@ -77,7 +77,7 @@ namespace mamba::fs
     std::filesystem::path normalized_separators(std::filesystem::path path);
 
     // Returns a UTF-8 string given a standard path.
-    std::string to_utf8(const std::filesystem::path& path);
+    std::string to_utf8(const std::filesystem::path& path, bool normalize_sep);
```

Comment:
I think it would be better with a struct `utf8Options` or something like that, which would have this boolean as member, with a default value  and passed here. That way, 
1) it's clearer at call site: `to_uft8(path, { .normilize_sep=true })`
2) it allows future option additions without breaking the function interface

Reply:
It's fine that way 👍🏼 (you can add the dot to the name so that it's ready to be uncommented in c++20)